### PR TITLE
Adding class DAO of Excursion

### PR DESCRIPTION
**Pull request by rajivvairo26**
please review this DAO class

### CODE REVIEW

File: TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    public Excursion findById(Long id) {
+        return eManager.find(Excursion.class, id);
+    }
+
+    @Override
+    public List<Excursion> findAll() {
+        return eManager.createQuery("select e from Excursion e", Excursion.class).getResultList();
+
+    }
+
```
Comment (filipceko) [inline TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java]: @Override annotation is missing

File: TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    @Override
+    public Excursion findById(Long id) {
+        return eManager.find(Excursion.class, id);
+    }
+
+    @Override
+    public List<Excursion> findAll() {
+        return eManager.createQuery("select e from Excursion e", Excursion.class).getResultList();
+
+    }
```
Comment (lyrithy) [inline TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java]: add one blank link

File: TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    @Override
```
Comment (lyrithy) [inline TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java]: one blank line should be added.

File: TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    public void update(Excursion excursion) {
+        eManager.merge(excursion);
+    }
+
+    @Override
+    public void remove(Excursion excursion) throws IllegalArgumentException {
+        eManager.remove(excursion);
+    }
+
```
Comment (lyrithy) [inline TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/dao/ExcursionDaoImpl.java]: blank line should be removed.

**Pull request merged**